core/src/crds_value.rs

```diff
@@ -96,13 +98,31 @@ impl Vote {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub struct Root {
+    pub from: Pubkey,
+    pub root: u64,
```

Comment:
The terminology in this PR doesn't make much sense to me. A `Root` is a struct, but a `Root::root` is a `u64`. There's a type alias for `Slot` - should it be used here? ...but is a `Slot` a slot or a slot height? The book says a root is a block or slot or maybe a block height or slot height. Lots more clarity all around would be very much appreciated.